pandas/core/frame.py

```diff
@@ -7555,7 +7555,9 @@ def nlargest(
         Italy     59000000  1937894      IT
         Brunei      434000    12128      BN
 
-        When using ``keep='all'``, all duplicate items are maintained:
+        When using ``keep='all'``, the number of element kept can go beyond n 
```

Comment:
I think it'd be better to highlight `n` here by wrapping it with backticks: ` ``n`` `.